### DISCUSSION TITLE
(cli_command) azuracast:internal:liquidsoap [example] [example]

### DISCUSSION THREAD

**Discussion by Marcioghiraldelli** [AzuraCast/AzuraCast#7145] [General]
I'm thinking about "playing" a little with the telnet or cli commands, but I'm a little lost on this part.
Firstly, here (cli_command) azuracast:internal:liquidsoap [example] [example] says that I can execute Liquidsoap commands, ok!
However, I have seen that using telnet, for example, I can start a remote playlist within a station's programming, and is this possible for CLI?

For example cli (cli_command) azuracast:internal:liquidsoap user station remote url

**Comment by BusterNeece** [comment]
@Marcioghiraldelli The command `azuracast:internal:liquidsoap` doesn't exist...are you requesting that it _should_ exist? You may want to submit that as a [feature request](https://features.azuracast.com/) instead.
